index.html

```diff
@@ -13,7 +14,18 @@
 
 <body>
     <div class="container">
+        <header>
+            <img src="C:\Users\Herbert\LambdaSchool\Sprint-Challenge--UI-Responsive\img\lambda-black.png" alt="Lambda Black Logo">
```

Comment:
Remember to use relative file paths for images and other assets that are stored within your project's file structure.